advisories/a82309.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Technical Advisory 82309
+advisory: A-82309
+summary: During or after an upgrade from CockroachDB v21.2.x to v22.1.0, existing changefeeds will stop emitting data.
+toc: true
+affected_versions: v22.1.0-alpha.1-v22.1.0
```

Comment:
should this be a comma between versions?

Reply:
This field shows up verbatim in the `Affected Versions` column here: https://deploy-preview-14012--cockroachdb-docs.netlify.app/docs/advisories/index.html